.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
```

Comment:
Shouldn't this exist under `.github/ISSUE_TEMPLATE`? Also, what kind of template is this? For bugs? I propose something like the following: https://github.com/umee-network/umee/blob/main/.github/ISSUE_TEMPLATE/bug-report.md

Reply:
I didn't happen to know we had a .github/ISSUE_TEMPLATE already if we did (I don't happen to see it), I can definite create a /ISSUE_TEMPLATE folder though. And this was originally supposed to be for generic issues, not specifically bugs.

Reply:
Yeah, we should have templates per type IMO (i.e. bugs, general proposal, features, etc...). Otherwise, having a template isn't all that useful.

Reply:
Alright, I'll make that then

Reply:
I think general proposals / features are done in a separate repo, and questions are done in our discord server, so I currently only implemented the bug reporter template, the generic issue template along with the option to still open blank issues. Let me know your thoughts :)

Reply:
I think bug report and general template is fine. I don't really understand why features are done in a separate repo, but I don't wanna bike shed here.